### DISCUSSION TITLE
Show link to repository home when in repository scope

### DISCUSSION THREAD

**Discussion by Jackenmen** [community/community#10163] [Other Feature Feedback, Questions, & Ideas] [answered]
Scenario:
- I go to https://cs.github.com
- I search for `org:linuxmint content:"xkb/"` and end up at https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Alinuxmint+content%3A%22xkb%2F%22
- I click on one of the repository names and end up at https://cs.github.com/linuxmint/ubiquity?q=org%3Alinuxmint%20content%3A%22xkb%2F%22
- On this page, I would like there to be a link that gets me to https://github.com/linuxmint/ubiquity

Workaround:
- Open one of the files in the results and end up at https://cs.github.com/linuxmint/ubiquity/blob/951f8cdca13d0ffc69eae0dc125c7b8e7060b44b/d-i/source/console-setup/man/keyboard.5.txt?q=org%3Alinuxmint+content%3A%22xkb%2F%22
- Click "view on github.com" and end up at https://github.com/linuxmint/ubiquity/blob/951f8cdca13d0ffc69eae0dc125c7b8e7060b44b/d-i/source/console-setup/man/keyboard.5.txt
- Click on the repository name and then end up at https://github.com/linuxmint/ubiquity

**Comment by colinwm** [comment, marked as answer]
We actually have a keyboard shortcut for this, just type `,` on https://cs.github.com/linuxmint/ubiquity?q=org%3Alinuxmint%20content%3A%22xkb%2F%22 and you'll jump to https://github.com/linuxmint/ubiquity. This can also be done when viewing individual files to jump to the github.com rendering of that file.

Sorry it's not very discoverable! If you want to see more keyboard shortcuts, you can use `?` to show all the available ones.

**Reply by eyalroth** [reply]
@colinwm The link refers to https://cs.github.com/linuxmint/ubiquity instead of https://github.com/linuxmint/ubiquity (no `cs` subdomain). Should I open a separate discussion for this?
